templates/partials/top-alert.html

```diff
@@ -1,12 +1,10 @@
 {% block content %}
 
-<div style = "display:table; overflow:hidden; display:table; height:7%; background-color:black; position:fixed; width:100%;">
-	<p class="mb6" style="color:white; display:table-cell; vertical-align:middle; text-align:center; opacity:100%">
+<div class="bg-black z3 pv6 fix w-full">
+	<p class="white t-center mv0 ph8">
                 Interested in developing on Urbit? Sign up for Hoon School! 
-		<a href="/hoonschool" style="color:white">Learn more</a>
+		<a href="/hoonschool" class="white">Learn more</a>
 	</p>
 </div>
-
-<div style="padding-bottom:70px"></div>
-
+<div class="pv14"></div>
```

Comment:
This is a good design change in that there needs more spacing between the top alert and the content, but its better practice to add margin to the top sibling (line 3). I also think `pv14` is a bit large and maybe something like `.mb4` would be better.

Reply:
Agreed about the spacing.  I did try the margin on the banner div but the issue is that it must have position:fixed so the margin isn't applied to the top of the body.  

I've done a bit of searching and can't seem to find a better solution.  Let me know if you might have any other thoughts

Reply:
I made some small design changes here:
I think fixed actually makes it quite bothersome/distracting for the user if they're trying to scroll and navigate to a different page. So I removed it.
I fixed the spacing here to be a bit less so the bar is more subtle.